client-gateway/bot/driver.py

```
@@ -1,32 +1,377 @@
+        else:
+            # New driver - needs registration
+            user_roles[chat_id] = {'role': 'driver', 'registered': False}
+            await update.message.reply_text(
+                "\u2705 Ви обрали роль: Таксист\n\n"
+                "\U0001F4DD Для початку роботи потрібно зареєструватися.\n"
+                "Натисніть кнопку нижче:",
+                reply_markup=driver_menu_unregistered()
+            )
+
+    async def start_registration(update: Update, context: ContextTypes.DEFAULT_TYPE):
+        chat_id = update.effective_chat.id
```

Flake8 (7.3.0)

[error] local variable 'chat_id' is assigned to but never used

(F841)